backend/module/feedback/feedback.go

```diff
@@ -0,0 +1,117 @@
+package feedback
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/any"
+	"github.com/uber-go/tally"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	feedbackv1cfg "github.com/lyft/clutch/backend/api/config/module/feedback/v1"
+	feedbackv1 "github.com/lyft/clutch/backend/api/feedback/v1"
+	"github.com/lyft/clutch/backend/module"
+)
+
+const (
+	Name = "clutch.module.feedback"
+)
+
+func New(cfg *any.Any, log *zap.Logger, scope tally.Scope) (module.Module, error) {
+	config := &feedbackv1cfg.Config{}
+	err := cfg.UnmarshalTo(config)
+	if err != nil {
+		return nil, err
+	}
+
+	m := &mod{
+		surveyMap: NewSurveyLookup(config.Origins),
+		logger:    log,
+		scope:     scope,
+	}
+
+	return m, nil
+}
+
+// SurveyLookup stores a map of the origin to survey mapping that are
+// provided in the feedback module config
+type SurveyLookup struct {
+	// key is the origin name
+	surveys map[string]*feedbackv1cfg.SurveyOrigin
+}
+
+type mod struct {
+	surveyMap SurveyLookup
+	logger    *zap.Logger
+	scope     tally.Scope
+}
+
+func (m *mod) Register(r module.Registrar) error {
+	feedbackv1.RegisterFeedbackAPIServer(r.GRPCServer(), m)
+	return r.RegisterJSONGateway(feedbackv1.RegisterFeedbackAPIHandler)
+}
+
+func (m *mod) GetSurveys(tx context.Context, req *feedbackv1.GetSurveysRequest) (*feedbackv1.GetSurveysResponse, error) {
+	results := make(map[string]*feedbackv1.Survey)
+
+	// this scenario shouldn't happen as the feedback config fields cannot be empty
+	if len(m.surveyMap.surveys) == 0 {
+		return nil, status.Errorf(codes.NotFound, "survey questions were not found")
```

Comment:
nit: add error log here and below

Reply:
ah thanks! forgot about error logs, will update.